compiler/src/formatting/format.re

```diff
@@ -3672,6 +3674,52 @@ and print_expression =
 
   expression_doc;
 }
+
+and group_expression = (expr: Parsetree.expression) => {
```

Comment:
would probably call this `is_grouped_access_expression`

Reply:
Agreed.  Renamed accordingly.